### PR TITLE
Fix refactor error in ECCV Adv Tutorial

### PR DESCRIPTION
**Pull request by Skylion007**
## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [X] Bug fix (non-breaking change which fixes an issue)
Fixes broken video function
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [X] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb [hunk 1]
```diff
@@ -1877,7 +1881,7 @@
     "\n",
     "def edit_wireframe_UVSphere(edit_template):\n",
     "    # @markdown Number of (line) rings. Must be larger or equal to 2 and multiple of 2.\n",
-    "    num_rings = 16  # @param {type:\"slider\", min:2, max:64, step:2}\n",
+    "    num_rings = 16  # @param {type:\"slider\", min:2, max:64, step:2} # noqa: F841\n",
     "    # @markdown Number of (line) segments. Must be larger or equal to 4 and multiple of 4.\n",
     "    num_segments = 40  # @param {type:\"slider\", min:4, max:64, step:4}\n",
     "    edit_template.num_segments = num_segments\n",
```
Comment (aclegg3) [inline examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb:1887]: Add `edit_template.num_rings = num_rings`.

Comment (aclegg3) [reply examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb:1887]: Correct me if wrong, @jturner65 🙂

Comment (jturner65) [reply examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb:1887]: yes that is correct

Comment (jturner65) [reply examples/tutorials/colabs/ECCV_2020_Advanced_Features.ipynb:1887]: So get rid of that  # noqa: F841
and add 
`edit_template.num_rings = num_rings`

**Pull request merged**